Gemfile.lock

```diff
@@ -242,96 +242,95 @@ GEM
       rest-client (>= 1.6.7)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
-    method_source (0.9.2)
-    mime-types (3.3)
+    method_source (1.0.0)
+    mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2019.1009)
-    mimemagic (0.3.3)
-    mini_magick (4.9.5)
+    mimemagic (0.3.4)
```

Comment:
### [mimemagic](https://github.com/minad/mimemagic)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.3.3 → 0.3.4

[_[change-log](https://github.com/minad/mimemagic/blob/master/CHANGELOG.md), [source-code](https://github.com/minad/mimemagic)_]





<details>
<summary>Commits</summary>

A change of **17** commits. See the full changes on [the compare page](https://github.com/minad/mimemagic/compare/v0.3.3...v0.3.4).

These are the first 10 commits:
- (f4afc9b) [Handle Office Open XML types for filed generated outside MSO…](https://github.com/minad/mimemagic/commit/f4afc9bac944abe284d74e481ee08009e10b1564)
- (487e3ca) [Add metadata uris](https://github.com/minad/mimemagic/commit/487e3ca0eedbba1b3bcf9b97970be78a63c0bcc7)
- (94f0a8c) [Merge pull request #68 from viraptor/metadata-uris](https://github.com/minad/mimemagic/commit/94f0a8cf4ef553c94de60a8d29b930e823043333)
- (96e8586) [README: Use SVG badges](https://github.com/minad/mimemagic/commit/96e85860d9aecd8cd345330933da2660b5d6ecba)
- (632e8c1) [README: Use GitHub Markdown code fences](https://github.com/minad/mimemagic/commit/632e8c1d90022d9aba8d6df76ee1fc0da15c6f59)
- (eb5ad01) [Travis: Use Bundler < 2](https://github.com/minad/mimemagic/commit/eb5ad01215e70c1f40c8e49b7fdf6ae4a25c9960)
- (b4b2a2f) [Merge pull request #70 from olleolleolle/patch-1](https://github.com/minad/mimemagic/commit/b4b2a2f80d8bd1690e003576f4f3cadd6dc99eb7)
- (72fed2b) [Merge pull request #71 from olleolleolle/patch-2](https://github.com/minad/mimemagic/commit/72fed2be6d4e3f440433fd70ebda4b9477dfd114)
- (317f9ff) [Merge pull request #72 from olleolleolle/patch-3](https://github.com/minad/mimemagic/commit/317f9ffedb547b1098386ce27c4ddae13dba0561)
- (5b4ea8e) [CI: rbx-3 in allow_failures, comment on Bundler](https://github.com/minad/mimemagic/commit/5b4ea8ee0a5fd145bdc52c945e97acdded18e5f9)


</details>